inst/tests/tests.Rraw

```diff
@@ -13030,7 +13030,7 @@ test(1915, fread(str, select = c(5, 1, 3)),
 
 # unit tests for #2169 which was already fixed in ec05f8b/src/fwrite.c#L59
 test(1916.1, fwrite(data.table(foo=c(NA, TRUE, FALSE)), logical01=TRUE, na='EMPTY', verbose=FALSE),
-             output=c("\"foo\"","EMPTY","1","0")) # this might need to be changed after #2964 will be solved
+             output=c("foo","EMPTY","1","0")) # changed as of resolving #2964
```

Comment:
Imo, this comment can be avoided if it's resolved (which is the case now), as it only makes sense to have such if it's on the back burner (unless it's a critical thing and we must require a reference)